client/src/api/block_builder/input_selection/helpers.rs

```diff
@@ -102,3 +105,23 @@ pub(crate) fn _sort_input_signing_data(inputs: Vec<InputSigningData>) -> crate::
 
     Ok(sorted_inputs)
 }
+
+/// Computes the minimum storage deposit amount that a basic output needs to have with an [AddressUnlockCondition] and
+/// optional [NativeTokens].
+pub fn minimum_storage_deposit_basic_output(
+    config: &RentStructure,
+    address: &Address,
```

Comment:
Do we still want to accept an address?
Initially it was required because the length of alias and nft addresses was shorter, but since that is no longer the case, it doesn't matter what address we have and would be easier to have it removed from the parameters

Reply:
Yeah I can do that

Reply:
Done